### DISCUSSION TITLE
Adding options for `targets/inst/pipelines/_targets_r.R`

### DISCUSSION THREAD

**Discussion by mattocci27** [ropensci/targets#771] [Ideas]
Currently, Target Markdown properly generates `_targets_r.R` only for `tar_make` but not for other `tar_make_xxx`. For example, even if we use `tar_make_clustermq(workers = 2)` inside the rmarkdown, the auto-generated `_targets_r.R` does not contain `options(clustermq.scheduler = "multiprocess")`.

It would be great if 1) we can select other templates of `_targets_r.R` inside from rmakdown or 2) we have an option not to overwrite `_targets_r.R` so that we can edit `_targets_r.R` later by hand.

**Comment by wlandau** [comment]
You can add a `{targets ..., tar_globals = TRUE}` code chunk to include HPC configuration like `future::plan()` and `options(clustermq.scheduler = "...", clustermq.template = "...')`. That way, the `_targets.R` from Target Markdown does not need to change. Example: https://github.com/wlandau/rmedicine2021-pipeline/blob/9acd1f9156157b5a3673f5d5ea59abb6450ab6d2/index.Rmd#L110-L112.

**Reply by mattocci27** [reply]
Oops, I didn't notice it. I will try it. Thank you so much for developing this package!
